tesseract-core/src/backend.rs

```diff
@@ -38,6 +38,10 @@ pub trait Backend {
             &query_ir.growth,
         )
     }
+
+    fn check_user(&self) -> Box<dyn Future<Item=(), Error= Error>> {
+        unimplemented!()
```

Comment:
One thing we might want to do by default here would be to log that this is unimplemented but maybe not error? thoughts?

Reply:
Yeah, that's definitely better than erroring out.